src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1997,10 +1997,13 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 				label = nls.localize('terminalTextBoxAriaLabel', "Terminal {0}", terminalId);
 			}
 			const screenReaderOptimized = this._accessibilityService.isScreenReaderOptimized();
-			if (screenReaderOptimized && isMacintosh) {
-				label += nls.localize('terminalTextBoxAriaLabelMacKeybinding', 'Press alt+f1 for terminal accessibility help');
+			if (!screenReaderOptimized) {
+				label += nls.localize('terminalScreenReaderMode', " To enter screen reader optimized mode, run the command Toggle Screen Reader Accessibility Mode.");
```

Comment:
It might be better to separate all by a ` ` like this?

```ts
labelParts.push(...)
labelParts.join('\n')
```

We don't want a leading space to get translated incorrectly.